keps/prod-readiness/sig-network/1860.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 1860
+alpha:
+  approver: "@wojtek-t"
```

Comment:
I didn't realize initially, but you didn't fill in PRR section.

Please copy the whole PRR section from the template:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#production-readiness-review-questionnaire

and fill-in at least the ones that are required for Alpha (others may be left empty for now).

And ping me once done :)

Reply:
Done. @wojtek-t